dist/vl-input-field.js

```diff
@@ -31,10 +33,21 @@ export class VlInputField extends vlFormValidation(nativeVlElement(HTMLInputElem
   }
 
   _dress() {
+    this._dressFormValidation();
+    this._dressPattern();
+  }
+
+  _dressFormValidation() {
     if (this.form) {
       this.setAttribute('data-vl-success-class', 'vl-input-field--success');
       this.setAttribute('data-vl-error-class', 'vl-input-field--error');
+      Object.assign(this, vlFormValidation);
       this.dress(this.form);
     }
   }
+
+  _dressPattern() {
+    Object.assign(this, vlPattern);
+    this.dress(this);
+  }
```

Comment:
Zie mijn comment in pattern wat betreft function name. Kan het dus zijn dat door die assign de functie wordt overschreven? Uiteindelijk werk het waarschijnlijk wel omdat dat je onmiddellijk dress aanroept.
Maar misschien niet zo proper?

Reply:
De mixins worden intern éénmalig aangeroepen. Het is bijgevolg niet nodig om beide dress functies tegelijk beschikbaar te hebben. Dit is gedekt via de unit testen.

Reply:
OK, maar heeft toch beetje een smell vind ik persoonlijk.

Nog een laatste die ik mij afvroeg waarom eigenlijk een mixin? In de functies zie ik op het eerste zicht niet waar je internal state, this, ... aanroept. Had je niet gewoon vlPattern.dress(this) kunnen doen? Of zie ik iets over het hoofd?

Reply:
Dat was ook mogelijk, maar conceptueel vind ik dat een input field validatie en pattern logica **heeft** vandaar dat ik voor een mixin gekozen heb. Nu is er ook de mogelijkheid om statefull logica toe te voegen als dat nodig zou zijn, wat flexibeler is dan een stateless singleton implementatie.